config/doxygen_config.in

```diff
@@ -2095,7 +2095,7 @@ INCLUDE_FILE_PATTERNS  =
 # recursively expanded use the := operator instead of the = operator.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-PREDEFINED             =
+PREDEFINED             = "Remember(c)=c"
```

Comment:
So this just replaces the text of the Remember() with the variable itself? :)

Reply:
This tells doxygen to always replace `Remember(c)` with `c` and is independent of the `DBC_LEVEL` setting.